scripts/openzfs-merge.sh

```diff
@@ -106,6 +109,9 @@ generate_desc() {
 #add description to commit
 add_desc_to_commit() {
 	git commit --amend -F "$TMP_FILE"
+	if [ $PUSH_NEW_BRANCH -ne 0 ]; then
```

Comment:
I made it for the case with manual merge, so after manual merge is done I use -g key to generate a valid description, so if it will be in merge function it won't run. But I understood you, will move it and do it more cleanly.